pkg/minikube/cluster/cluster.go

```diff
@@ -371,12 +373,30 @@ func GetHostDockerEnv(api libmachine.API) (map[string]string, error) {
 }
 
 // GetHostLogs gets the openshift logs of the host VM.
-func GetHostLogs(api libmachine.API) (string, error) {
+// If follow is specified, it will tail the logs
+func GetHostLogs(api libmachine.API, follow bool) (string, error) {
 	host, err := CheckIfApiExistsAndLoad(api)
 	if err != nil {
 		return "", err
 	}
-	s, err := host.RunSSHCommand(logsCmd)
+
+	cmdToRun := logsCmd
+	if follow {
+		cmdToRun = logsCmdFollow
+		c, err := host.CreateSSHClient()
+		if err != nil {
+			return "", err
+		}
+
+		err = c.Shell(cmdToRun)
+		if err != nil {
+			return "", errors.Wrap(err, "Error creating ssh client")
+		}
+
+		return "", err
```

Comment:
Shouldn't this `return "", nil`, I understand if there is no err then err will be `nil` but here it would be good to put it explicit.